torch/csrc/autograd/autograd.cpp

```diff
@@ -1,3 +1,9 @@
+#ifndef AT_PER_OPERATOR_HEADERS
+#include <ATen/Functions.h>
+#else
+#include <ATen/ops/ones_like.h>
+#endif
+
```

Comment:
Thanks for the clean up!

nit: you might still want `autograd.h` to be the first import here just to follow convention

Reply:
fixed